Sources/DangerFixtures/DangerDSLResources/DSLBitBucketCloudJSON.swift

```diff
@@ -0,0 +1,569 @@
+                "type": "commit",
+                "links": {
+                  "self": {
+                    "href": "https://api.bitbucket.org/2.0/repositories/f-meloni/danger-kotlin/commit/9ab48765728b"
+                  },
+                  "html": {
+                    "href": "https://bitbucket.org/f-meloni/danger-kotlin/commits/9ab48765728b"
+                  }
+                }
+              },
+              "repository": {
+                "links": {
+                  "self": {
+                    "href": "https://api.bitbucket.org/2.0/repositories/f-meloni/danger-kotlin"
+                  },
+                  "html": {
+                    "href": "https://bitbucket.org/f-meloni/danger-kotlin"
+                  },
+                  "avatar": {
+                    "href": "https://bytebucket.org/ravatar/%7B37c7bd5b-887c-4450-99cb-3cab9c456f3d%7D?ts=default"
+                  }
+                },
+                "type": "repository",
+                "name": "danger-kotlin",
+                "full_name": "f-meloni/danger-kotlin",
+                "uuid": "{37c7bd5b-887c-4450-99cb-3cab9c456f3d}"
+              },
+              "branch": {
+                "name": "Franco-Meloni/readmemd-edited-online-with-bitbucket-1563114726802"
+              }
+            },
+            "state": "OPEN",
+            "author": {
+              "display_name": "Franco Meloni",
+              "uuid": "{bd1991e4-a3ed-45b2-be38-acea659650f1}",
+              "links": {
+                "self": {
+                  "href": "https://api.bitbucket.org/2.0/users/%7Bbd1991e4-a3ed-45b2-be38-acea659650f1%7D"
+                },
+                "html": {
+                  "href": "https://bitbucket.org/%7Bbd1991e4-a3ed-45b2-be38-acea659650f1%7D/"
+                },
+                "avatar": {
+                  "href": "https://secure.gravatar.com/avatar/3d90e967de2beab6d44cfadbb4976b87?d=https%3A%2F%2Favatar-management--avatars.us-west-2.prod.public.atl-paas.net%2Finitials%2FFM-1.png"
+                }
+              },
+              "nickname": "f-meloni",
+              "type": "user",
+              "account_id": "557058:cac1aa4b-db5e-4c3a-8ba3-c4880b498de8"
+            },
+            "date": "2019-07-14T14:32:09.461209+00:00"
+          },
+          "pull_request": {
+            "type": "pullrequest",
+            "id": 1,
+            "links": {
+              "self": {
+                "href": "https://api.bitbucket.org/2.0/repositories/f-meloni/danger-kotlin/pullrequests/1"
+              },
+              "html": {
+                "href": "https://bitbucket.org/f-meloni/danger-kotlin/pull-requests/1"
+              }
+            },
+            "title": "README.md edited online with Bitbucket"
+          }
+        }
+      ]
+    },
+    "settings": {
+      "github": {
+        "accessToken": "NO_T...",
+        "additionalHeaders": {}
+      },
+      "cliArgs": {}
+    }
+  }
+}
+"""
```

Comment:
<!--
  0 failure: 
  1 warning:  File should conta...
  
  
  DangerID: danger-id-Danger;
  File: Sources/DangerFixtures/DangerDSLResources/DSLBitBucketCloudJSON.swift;
  Line: 569;
-->

- :warning: File should contain 400 lines or less: currently contains 569 (`file_length`)